modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/exp/agg/AccumulatorsFactory.java

```diff
@@ -214,6 +214,11 @@ private WrapperPrototype(AggregateCall call) {
             List<RelDataType> inTypes = SqlTypeUtil.projectTypes(inputRowType, call.getArgList());
             List<RelDataType> outTypes = accumulator.argumentTypes(ctx.getTypeFactory());
 
+            if (call.getArgList().size() > outTypes.size()) {
```

Comment:
Is the check is necessary? Does the validator pass count with multiple arguments?
Is the check valid for other accumulators? e.g. LIST_AGG, etc?